core/inx/params.go

```diff
@@ -0,0 +1,23 @@
+package inx
+
+import (
+	flag "github.com/spf13/pflag"
+
+	"github.com/gohornet/hornet/pkg/node"
+)
+
+const (
+	// CfgINXAddress the INX address to which to connect to.
+	CfgINXAddress = "inx.address"
```

Comment:
Should be `"inx.bindAddress"`

Reply:
lol, right.
My fault.
But then the template config file is also wrong :runner: